Formula/rgbds.rb

```diff
@@ -21,24 +21,20 @@ class Rgbds < Formula
   end
 
   depends_on "bison" => :build
+  depends_on "cmake" => :build
   depends_on "pkg-config" => :build
   depends_on "rust" => :build
   depends_on "libpng"
 
   resource "rgbobj" do
-    url "https://github.com/gbdev/rgbobj/archive/refs/tags/v0.1.0.tar.gz"
-    sha256 "359a3504dc5a5f7812dfee602a23aec80163d1d9ec13f713645b5495aeef2a9b"
-
-    # Fix support for clap 3.2+. Remove in the next release.
-    # Issue ref: https://github.com/gbdev/rgbobj/issues/4
-    patch do
-      url "https://github.com/gbdev/rgbobj/commit/b4e1fe42dbc297d67d67ed17004a3f6956de199f.patch?full_index=1"
-      sha256 "27aceed0020c7561e8259308df0868d19e0b7f21ad5e0f2a389d591e8b60027d"
-    end
+    url "https://github.com/gbdev/rgbobj/archive/refs/tags/v0.2.1.tar.gz"
+    sha256 "3d91fb91c79974700e8b0379dcf5c92334f44928ed2fde88df281f46e3f6d7d1"
   end
 
   def install
-    system "make", "install", "PREFIX=#{prefix}", "mandir=#{man}"
+    system "cmake", "-S", ".", "-B", "build", "-DCMAKE_BUILD_TYPE=Release"
+    system "cmake", "--build", "build", "-j"
+    system "cmake", "--install", "build", "--prefix", prefix.to_s, "--strip"
```

Comment:
This just builds out fine.
```suggestion
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
```

Reply:
The `-j` is to allow a parallel build, which I don't believe is enabled by default?

Reply:
> The `-j` is to allow a parallel build, which I don't believe is enabled by default?

I think we have that enable by default. (`ENV.deparallelize` is to opt out, https://github.com/Homebrew/brew/blob/master/Library/Homebrew/formula_creator.rb#L155)